utils/data_preprocessing.py

```diff
@@ -259,12 +269,15 @@ def clean_digits(content: List[str]) -> List[str]:
 
 
 def clean_short_words(content: List[str]) -> List[str]:
```

Comment:
Then change the name to `clean_short_and_long_words`. Not the best name, maybe you find smth better.

Reply:
okay sure I can rename that